src_erl/erlBridge/nerlTests.erl

```diff
@@ -28,10 +28,8 @@ nerltest_print(String) ->
 
 test_envelope(Func, TestName, Rounds) ->
       nerltest_print(nerl:string_format("~p test starts for ~p rounds",[TestName, Rounds])),
-      Tic = nerl:tic(),
-      Func(Rounds),
-      {Diff, TimeUnit} = nerl:toc(Tic),
-      nerltest_print(nerl:string_format("Elapsed: ~p~p",[Diff,TimeUnit])), ok.
+      {TimeTookMicro, _RetVal} = timer:tc(?MODULE, Func, [Rounds]),
```

Comment:
@halfway258 
This might cause an issue when rebase with master. 
Please review it carefully, I suggest pulling master into local master branch, look at the output that Nerltests.sh generates and then perform the rebase and change this branch **to achieve the same output. **